Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -206,8 +206,8 @@ class RenderStyle {
 
     const PseudoStyleCache* cachedPseudoStyles() const { return m_cachedPseudoStyles.get(); }
 
-    const CustomPropertyValueMap& inheritedCustomProperties() const { return m_rareInheritedData->customProperties->values; }
-    const CustomPropertyValueMap& nonInheritedCustomProperties() const { return m_rareNonInheritedData->customProperties->values; }
+    const StyleCustomPropertyData& inheritedCustomProperties() const { return m_rareInheritedData->customProperties.get(); }
+    const StyleCustomPropertyData& nonInheritedCustomProperties() const { return m_rareNonInheritedData->customProperties.get(); }
     const CSSCustomPropertyValue* customPropertyValue(const AtomString&) const;
 
     void deduplicateCustomProperties(const RenderStyle&);
```

Comment:
Per offline conversation, we still want to call `deduplicateCustomProperties`, since we still can (and still want to avoid) setting a new `StyleCustomPropertyData` object with equal values.